pkg/route/ingress/ingress.go

```diff
@@ -370,6 +372,27 @@ func (c *Controller) sync(key queueKey) error {
 		return err
 	}
 
+	// If the ingress specifies an ingressclass and the ingressclass does
+	// not specify OpenShift Router as its controller, ignore the ingress.
+	var ingressClassName *string
+	if v, ok := ingress.Annotations["kubernetes.io/ingress.class"]; ok {
+		ingressClassName = &v
+	} else if ingress.Spec.IngressClassName != nil {
```

Comment:
```suggestion
	} else {
```

No need to check for nil twice, right?

Reply:
Right.  Thanks!